package-lock.json

```diff
@@ -12592,7 +12592,8 @@
       "version": "1.0.4",
```

Comment:
Please don't include this file in the PR 😇

Reply:
ok , reverted to the previous version